src/FormGroup.js

```diff
@@ -30,16 +30,17 @@ const FormGroup = (props) => {
     inline,
     floating,
     tag: Tag,
+    switch: switchProp,
```

Comment:
Can you leave this as `switch` instead of `switchProp`?

Reply:
Unfortunately not - doing so means you end up with ambiguity between the `switch` variable and JavaScript  `switch`/`case` statement - I can call it something else if you'd prefer, e.g. `_switch`?

Reply:
Hello, It would be nice to see this ugly error message fixed.
Maybe a comment to explain the variable name would be good to approve this merge? 
The other way is to keep the props.switch notation?
Thanks

Reply:
We can't leave it as `props.switch` - destructuring the prop like this fixes the issue - without it `switch` remains in the `attributes` object that is passed through to the underlying control.

As I say, I'm very happy to use whatever name is deemed appropriate, but as far as I know this PR hasn't even been looked at by any of the maintainers, so I don't think that's what's causing it to be left unmerged.

Reply:
workaround: you can remove the switch prop from FormGroup and replace with the bootstrap classname:
` <FormGroup className="form-check form-switch">`

Reply:
Had forgotten to respond, the PR looks good 👍